src/components/home/SectionThree.tsx

```diff
@@ -13,9 +14,10 @@ const HouseCard = () => {
   const toggleLike = () => {
     setLiked(!liked);
   };
+
   return (
-    <Card className='w-[350px] rounded-xl flex flex-col gap-5 pb-5 '>
-      <header className='relative'>
+    <Card className='w-full sm:w-[350px] rounded-xl flex flex-col gap-5 pb-5 px-4'>
```

Comment:
Can you separate the card out as components, make it as general as possible. create a data for render so that thế can implement  the API .

Reply:
I have separated the component as you said and created a mock houses data which is house.ts in a data folder and run as an array to test the house card component